data/remnant/remnant outfits.txt

```diff
@@ -580,11 +599,16 @@ outfit "Smelter-Class Steering"
 	"turn" 1980.0
 	"turning energy" 5.21
 	"turning heat" 6.2
+	"slowing resistance" 0.4
```

Comment:
Still seems way too high to me (Smelter thrusters + steering = bleed all slowing damage from 4 Inhibitors in 3 frames, Crucible thrusters + steering = bleed all slowing from 4 inhibitors in 9 frames), and still also don't want it.

Let's just drop slowing resistance on all the engines. Then just so the steering aren't losing out by not having reverse thrust like the thrusters do, how about we instead bump their turn rates a bit? An increase in turn for all steering by just 10% (no change to any other stats) would mean that Remnant steering...
* would be the BEST purchasable steering in terms of heat production vs turn rate, beat only by Pug steering, which can not be purchased.
* would become a little better on energy drained vs turn rate, as right now they're already the worst engines on that front, but this change would make them a little more bearable.
* would have the best small steering, best medium steering, and second best large steering in terms of turn rate vs mass.

Reply:
OK, I'll take a look at making some changes tomorrow. Would it be possible for you to share that spreadsheet you were showing in the livestream in your folder of spreadsheets? 

I still want them to have some slow resistance, but I also want it to be reasonable. And your numbers there make it sound like it is still too high.

Reply:
Alright. Reduced the slow resistance to 10% of those values. So assuming your math is correct (which I assume it is), bleeding that slow will take 90 frames instead of 9. And increased the steering values by 5%.